src/popup/components/NameItem.vue

```diff
@@ -193,16 +208,23 @@ export default defineComponent({
     const { activeAccount } = useAccounts();
     const { t } = useI18n();
     const { topBlockHeight } = useTopHeaderData({ store });
+    const {
+      setAutoExtend,
+      updatePointer,
+      getName,
+      getDefaultName,
+      setDefault,
+    } = useAeNames({ store });
+    const { aeActiveNetworkSettings } = useAeNetworkSettings();
+    const { fetchRespondChallenge } = useAeSdk({ store });
 
     const expand = ref(false);
     const newPointer = ref<string>('');
     const showInput = ref(false);
     const error = ref(false);
     const pointerInput = ref();
 
-    const getDefaultName = store.getters['names/getDefault'] as (a?: string) => string | undefined;
-
-    const nameEntry = computed<IName | null>(() => store.getters['names/get'](props.name));
+    const nameEntry = computed((): IName | undefined => getName(props.name));
```

Comment:
As this computed simply returns the function I suggest to remove the typings as it is inherited from `getName`.